HW_Multithreading/src/MyThread.java

```diff
@@ -0,0 +1,22 @@
+public class MyThread implements Runnable {
+    private Counter count;
+   // Thread t;
+
+    public MyThread(Counter count) {
+        this.count = count;
+       /* t=new Thread();
+        t.start();*/
+    }
+
+
+    @Override
+    public void run() {
+        for (int i = 0; i <10 ; i++) {
+            count.increment();
```

Comment:
This is also wrong: one thread (t1) should increment, second, t2, should decrement.